Parser/YSI6SeriesParse.m

```diff
@@ -1,323 +1,329 @@
-      % ug/L == mg/m^3
-      case 'chlorophyll'
-        sample_data.variables{end}.name           = 'CPHL';
-        sample_data.variables{end}.typeCastFunc   = str2func(netcdf3ToMatlabType(imosParameters(sample_data.variables{end}.name, 'type')));
-        sample_data.variables{end}.data           = sample_data.variables{end}.typeCastFunc(field');
-        sample_data.variables{end}.comment        = ['Artificial chlorophyll data '...
-            'computed from bio-optical sensor raw counts measurements. The '...
-            'fluorometre is equipped with a 470nm peak wavelength LED to irradiate and a '...
-            'photodetector paired with an optical filter which measures everything '...
-            'that fluoresces in the region above 630nm'...
-            'Originally expressed in ug/l, 1l = 0.001m3 was assumed.'];
-        
-      % relative fluorescence unit
-      %case 'chlorophyllRFU'
-      %  sample_data.variables{end}.name = 'CPHL_RFU';
-        
-      case 'latitude'
-        sample_data.variables{2}.data             = sample_data.variables{2}.typeCastFunc(field');
-        
-      case 'longitude'
-        sample_data.variables{3}.data             = sample_data.variables{3}.typeCastFunc(field');
-        
-      case 'turbidity'
-        sample_data.variables{end}.name           = 'TURB';
-        sample_data.variables{end}.typeCastFunc   = str2func(netcdf3ToMatlabType(imosParameters(sample_data.variables{end}.name, 'type')));
-        sample_data.variables{end}.data           = sample_data.variables{end}.typeCastFunc(field');
-        sample_data.variables{end}.comment        = 'Turbidity from 6136 sensor.';
-        
-      % % saturation
-      case 'odo'
-        sample_data.variables{end}.name           = 'DOXS';
-        sample_data.variables{end}.typeCastFunc   = str2func(netcdf3ToMatlabType(imosParameters(sample_data.variables{end}.name, 'type')));
-        sample_data.variables{end}.data           = sample_data.variables{end}.typeCastFunc(field');
-        sample_data.variables{end}.comment        = ...
-          'Dissolved oxygen saturation from ROX optical sensor.';
-        
-      % mg/l
-      case 'odo2'
-        sample_data.variables{end}.name           = 'DOXY';
-        sample_data.variables{end}.typeCastFunc   = str2func(netcdf3ToMatlabType(imosParameters(sample_data.variables{end}.name, 'type')));
-        sample_data.variables{end}.data           = sample_data.variables{end}.typeCastFunc(field');
-        sample_data.variables{end}.comment        = ...
-          'Dissolved oxygen from ROX optical sensor.';
+
+    % only one file supported currently
+    filename = filename{1};
+
+    if ~exist(filename, 'file'), error([filename ' does not exist']); end
+
+    % read in the whole file into 'data'
+    fid = -1;
+
+    try
+        fid = fopen(filename, 'rb');
+        data = fread(fid, inf, '*uint8');
+        fclose(fid);
+    catch e
+        if fid ~= -1, fclose(fid); end
+        rethrow(e);
```

Comment:
This commit is about converting any CHLF/CHLU into CPHL but you've made other changes. It would be clearer if they were in another commit and you explainwhat are the changes for this commit.

Reply:
as above, the diff is confusing because of default options and the indents. There is no change.